docs/_posts/2024-01-15-sitecore-10-problems-and-solutions.md

```diff
@@ -0,0 +1,197 @@
+---
+title:  "Sitecore 10 Problems And Solutions"
+date:   2024-01-08 12:00:00 +0100
```

Comment:
Otherwise it will get published too early
```suggestion
date:   2024-01-15 12:00:00 +0100
```

Reply:
@tutkucan I will commit this before the merge to master tomorrow, afterward, we can set it back on the develop branch :)

Reply:
thank you ramo